package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```

Comment:
Seems to be added by accident?

Reply:
WTH? Yes, will remove again. Good catch! :)

Reply:
Fixed via https://github.com/wp-cli/media-command/pull/115/commits/ba9f3581e730a7771bce32f07148240cf8dc6637